packages/LSOnline/modules/player/playerEvents.js

```diff
@@ -65,10 +55,46 @@ mp.events.add({
   },
 
   playerChat: (player, text) => {
+    if (!player.isLogged) {
+      return player.kick();
+    }
+
     if (player.brutallyWounded) {
       return player.call('actionDone', ['Nie możesz tego teraz użyć!', 'Twoja postać jest nieprzytomna!']);
     }
 
     mp.players.broadcastInRange(player.position, 25, player.dimension, `${player.name} mówi: ${text}`);
   }
 });
+
+const createQuitLabel = (player, exitType) => {
```

Comment:
Function should be extracted somewhere else

Reply:
look at 53f3925.